packages/bundler-plugin-core/src/index.ts

```diff
@@ -15,9 +14,10 @@ import {
   getDependencies,
   getPackageJson,
   parseMajorVersion,
-  stringToUUID,
 } from "./utils";
 
+import { v4 as uuidV4 } from "uuid";
```

Comment:
l: What's our minimum supported Node version again? 😅 
If it is >14 we should be good to use [`randomUUID()`](https://stackoverflow.com/a/67624847), right? 
(but overall logaf-super-l)